internal/namespace/typesystem.go

```diff
@@ -474,6 +475,97 @@ func (nts *TypeSystem) typeSystemForNamespace(ctx context.Context, namespaceName
 	return BuildNamespaceTypeSystem(nsDef, nts.lookupNamespace)
 }
 
+// Searches all computed user set relations given a tupleToUserSet relation
```

Comment:
This needs a better name and description. What, exactly, is this doing? Finding all right hand sides of arrows for the relation?

Reply:
I renamed it to `ResolveArrowRelations(string)`

Is it ok ?

Thanks